tests/sentry/api/bases/test_user.py

```diff
@@ -39,6 +41,35 @@ def test_allows_active_superuser(self):
             self.superuser_request, None, self.normal_user
         )
 
+        with self.settings(SENTRY_SELF_HOSTED=False):
+            assert self.user_permission.has_object_permission(
+                self.superuser_request, None, self.normal_user
+            )
+
+    @override_settings(SENTRY_SELF_HOSTED=False)
+    @with_feature("auth:enterprise-superuser-read-write")
+    def test_active_superuser_read(self):
+        # superuser read can hit GET
+        request = self.make_request(user=self.superuser_user, is_superuser=True, method="GET")
+        request.access = self.create_request_access()
+        assert self.user_permission.has_object_permission(request, None, self.normal_user)
+
+        # superuser read cannot hit POST
+        request.method = "POST"
+        assert not self.user_permission.has_object_permission(request, None, self.normal_user)
+
+    @override_settings(SENTRY_SELF_HOSTED=False)
+    @with_feature("auth:enterprise-superuser-read-write")
+    def test_active_superuser_write(self):
+        # superuser write can hit GET
+        request = self.make_request(user=self.superuser_user, is_superuser=True, method="GET")
+        request.access = self.create_request_access(permissions=["superuser.write"])
```

Comment:
if i don't set this explicitly, i get `AttributeError: 'HttpRequest' object has no attribute 'access'`. for some reason it doesn't set the request.access object in the overridden make_request function in `DRFPermissionTestCase` even though it does in other test classes that don't subclass `DRFPermissionTestCase`

Reply:
on second thought i think this is because we don't call `self.determine_access` in `UserPermission`